### ISSUE TITLE
Error: TypeError: Cannot read property 'addEventListener' of undefined

### ISSUE THREAD

**Issue by johanbove** [beakerbrowser/beaker#1521]
Operation System: Windows 10
Beaker version: 0.8.9

View: "Currently Seeding"

Opening the details for a seeding site, the view opens, but then the error message is displayed.
The sites in "Your Archives" do not have this issue.

```
view.js:4127 Load error TypeError: Cannot read property 'addEventListener' of undefined
    at ProgressMonitor.setup (view.js:8007)
    at LibraryDatArchive.startMonitoringDownloadProgress (view.js:7599)
    at setup$$1 (view.js:4069)
```

<img width="1071" alt="Annotation 2020-03-13 161002" src="https://user-images.githubusercontent.com/922765/76633880-b4e0c680-6545-11ea-8946-7685a0dc9dc5.png">

**Comment by pfrazee** [comment]
Interesting, thanks for reporting. Looking into it right now.

**Comment by pfrazee** [comment]
Pushed a fix. Going to release 0.8.10.

**Comment by pfrazee** [comment]
Fixed in `0.8.10`

**Comment by johanbove** [comment]
Fix confirmed. Thanks for the fast resolution!
